zboss/doc/zboss_configuration.rst

```diff
@@ -33,4 +33,5 @@ To enable additional features in the ZBOSS libraries, you can use the following
   This option is enabled by default in the :ref:`Zigbee NCP sample <nrf:zigbee_ncp_sample>`.
   This option uses a production version of ZBOSS that has not been certified.
 * :kconfig:`CONFIG_ZIGBEE_GP_CB` - With this option enabled, the application can support the Green Power Combo feature, which implements the basic set of Green Power Proxy and Green Power Sink functionalities within a single device.
+  This option requires access to ZBOSS stack sources to be enabled.
```

Comment:
What does this mean, in practice?

Reply:
Green Power Combo Basic functionality can be used only if zigbee samples are built with ZBOSS stack from sources

Reply:
OK, can you phrase this sentence in this manner? Thanks.

Reply:
done!